packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -230,7 +235,7 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   }
 
   /* @conditional-compile-remove(video-background-effects) */
-  if (useIsParticularSidePaneOpen('videoeffects') && props.errorBarProps) {
+  if ((useIsParticularSidePaneOpen('videoeffects') || !isCameraOn) && props.errorBarProps) {
```

Comment:
We haven't used this anywhere in the codebase yet, but instead of hiding the error when the camera is off, could we clear the error in the the handler for toggleCamera using `newClearCallErrorsModifier`?
![image](https://github.com/Azure/communication-ui-library/assets/2684369/5e048a41-46ea-493a-90a4-73161c3e849d)

Honestly no idea if that works just curious if its possible.

Reply:
The way its currently implemented, does the error re-show when the camera is turned back on?

Reply:
is this supposed to be !isCameraOn?
The title indicates that it should dismiss when camera is off, but this is settings an activemessage when camera is off.

I see that in videoEffectsPane.tsx line 149 new change it does what I expected.
Correct me if I'm wrong here.

Reply:
- For 1st suggestion, using `newClearCallErrorsModifier`, this modifier is not actually exposed in the calling stateful layer. If we are planning to expose this modifier, it would be called in the stateful layer mostly in the call context by introducing new set errors method. We can trigger this method whenever we want to delete certain errors. I can pursue this as a separate task if we feel strongly about this approach.

- For 2nd question, as part of the second change for auto-dismiss(https://github.com/Azure/communication-ui-library/pull/3160). Whenever camera is turned on, it will reset a new active effect and if the error is generated again it would be shown and otherwise no error will be shown.